ext/standard/info.c

```diff
@@ -202,6 +202,9 @@ static ZEND_COLD void php_print_gpcse_array(char *name, uint32_t name_length)
 			} else {
 				php_info_print(" => ");
 			}
+			if (Z_TYPE_P(tmp) == IS_REFERENCE) {
+				ZVAL_DEREF(tmp);
+			}
```

Comment:
Just ZVAL_DEREF is enough, it already contains the reference check.

Reply:
Ah, right!